configure.ac

```diff
@@ -236,6 +236,24 @@ if test x$enable_bip70 != xno; then
   AC_MSG_ERROR([BIP70 is no longer supported!])
 fi
 
+AC_ARG_WITH([libmultiprocess],
+  [AS_HELP_STRING([--with-libmultiprocess=yes|no|auto],
+  [Build with libmultiprocess library. (default: detect is auto, i.e. detect with pkg-config)])],
```

Comment:
It seems now the first "detect" is unneeded, no?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/16367#discussion_r388529793

> It seems now the first "detect" is unneeded, no?

Thanks, fixed